web/client/components/misc/enhancers/basic.js

```diff
@@ -0,0 +1,62 @@
+const {compose, withStateHandlers} = require('recompose');
```

Comment:
move this in a combobox folder in misc, together with all comboboxes (Paged, Controlled...)